lametro/management/commands/generate_tag_analytics.py

```diff
@@ -0,0 +1,37 @@
+import csv
+from tqdm import tqdm
+from django.core.management.base import BaseCommand
+
+from lametro.models import LAMetroBill
+
+
+class Command(BaseCommand):
+
+    help = "This command produces a CSV file that lists each Board Report's tags."
+
+    def handle(self, *args, **options):
+
+        with open('tag_analytics.csv', 'w', newline='') as f:
+            writer = csv.writer(f, delimiter=',')
+            writer.writerow(
+                ('File ID',
+                 'Identifier'
+                 'Title',
+                 'Last Action Date',
+                 'Tag Classification',
+                 'Tag')
+            )
+
+            for bill in tqdm(LAMetroBill.objects.all()):
+                if not bill.board_report:
+                    continue
+
```

Comment:
P.s., Do we need this check?

Reply:
Hm, I guess we don't since the instances returned by `LAMetroBillManager` should all have valid `board_report`s.